src/modules/network/utils/getNetwork.test.js

```diff
@@ -8,6 +8,9 @@ describe('Utils: getNetwork', () => {
   describe('getNetworksList', () => {
     const response = [
       { label: 'Mainnet', name: 'mainnet' },
+      { label: 'Alphanet', name: 'alphanet' },
+      { label: 'Betanet', name: 'betanet' },
```

Comment:
just a clarification for a chain other then main chain would they always have this network types